### ISSUE TITLE
Day/night time based on current location

### ISSUE THREAD

**Issue by niks255** [Yanndroid/DualWallpaper#13]
Would be really cool

**Comment by Yanndroid** [comment]
Do you mean when you're at home or work? Constantly scanning for your current location can significantly reduce your battery life. A more efficient approach would be to check for a Wi-Fi network instead. For example, when you arrive at home, your device can automatically connect to your home Wi-Fi, and my app can then listen for this event to change the wallpaper.

**Comment by niks255** [comment]
> Do you mean when you're at home or work? Constantly scanning for your current location can significantly reduce your battery life. A more efficient approach would be to check for a Wi-Fi network instead. For example, when you arrive at home, your device can automatically connect to your home Wi-Fi, and my app can then listen for this event to change the wallpaper.

No, i mean receiving sunset and sunrise times for current location and using those values as day/night times. Updating those values every 12 or 24 hours or so should be sufficient and not heavy on battery.
There are services that provide sunset and sunrise times for given location, for example https://sunrise-sunset.org/api
